blazor/file-manager/how-to/add-custom-tool-bar.md

```diff
@@ -9,9 +9,9 @@ documentation: ug
 
 # Adding Custom Item to Toolbar in Blazor FileManager Component
 
-The custom toolbar items can be added and customized using the `ToolbarSettings` API and `ToolbarCreated` events.
+The custom toolbar items can be added and customized using the `ToolbarSettings` API and `ToolbarItems` property.
```

Comment:
The custom toolbar items can be added and customized using the `ToolbarSettings` API and `FileManagerCustomToolbarItem` property.

Reply:
Changes will update in next commit.